lab-yunjoo/ref-vs-val/pass-by-reference.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
```

Comment:
You should not Include code in the pull request that is not related to the assignment, You will want to remove this file from the project